src/main/resources/ESH-INF/thing/controller_serial.xml

```diff
@@ -17,6 +17,13 @@
             <channel id="serial_cse" typeId="serial_cse" />
         </channels>
 
+        <properties>
+            <property name="modelId"></property>
```

Comment:
Is there a reason to add the properties here? I guess it doesn't hurt, but I'm wondering why do it?

Reply:
Hi @cdjackson! Is your question only related to the "modelId" property or to all of them? I have created the PR because I would like to make use of the Home ID property in https://github.com/openhab/org.openhab.binding.zwave/pull/1388: reading the value through Python in the rfc2217 gateway (https://github.com/bodiroga/rfc2217-gateway) and announcing it using mDNS. Having the Home ID set as the representation property, the thing could be updated automatically if the gateway changes its IP address. I would update #1388 if this PR gets merged, sure. Anyway, I can delete the "modelId", "manufacturerId" and "nodeId" if you consider them superfluous, but the "homeId" property is essential to :wink:

Reply:
My question is really - why did you need to put them in the XML file? I guess it doesn't hurt, but I don't list other properties, so it makes it a bit non-standard. I guess we could alternatively add other properties as well to make it standard - I'd just prefer one or the other and I'm not sure I see the point of adding it?

Reply:
Hi Chris! I added them following the developers documentation: https://www.openhab.org/docs/developer/bindings/thing-xml.html#properties. But reading the following sentence again: ```Note that the representation property is normally not listed in the properties part of the Thing type XML, as this part contains static properties, that are the same for all instances of this Thing type```, I think that you are totally right. As the properties are dynamic and not the same for all serial Things (we have different serial controllers), I will remove them from the XML file, leaving the ```representation-property``` line alone. Sorry for my error :wink: